### PR TITLE
Adding 'buttonTopPosition' prop to control button's y position

### PR DESCRIPTION
**Pull request by Inbal-Tish**

### CODE REVIEW

File: src/expandableCalendar/calendarProvider.js [hunk 1]
```diff
@@ -31,6 +32,8 @@ class CalendarProvider extends Component {
     onMonthChange: PropTypes.func,
     /** whether to show the today button */
     showTodayButton: PropTypes.bool,
+    /** The button top position */
+    buttonTopPosition: PropTypes.number,
```
Comment (ethanshar) [inline src/expandableCalendar/calendarProvider.js]: shouldn't it be bottomPosition? 
also I would name the prop `todayButton...Position`

File: src/expandableCalendar/calendarProvider.js [hunk 2]
```diff
@@ -119,7 +122,7 @@ class CalendarProvider extends Component {
       const isToday = today === date;
       
       Animated.spring(this.state.buttonY, {
-        toValue: isToday ? 65 : -65,
+        toValue: isToday ? TOP_POSITION : -this.props.buttonTopPosition || -TOP_POSITION,
```
Comment (ethanshar) [inline src/expandableCalendar/calendarProvider.js]: What happens when `isToday` is false that you need a different value?

Comment (Inbal-Tish) [reply src/expandableCalendar/calendarProvider.js]: I need it to get out of the frame. It won't with every value so I'm keeping my own value

File: src/expandableCalendar/calendarProvider.js (hunk 1 above)
Comment (Inbal-Tish) [reply src/expandableCalendar/calendarProvider.js]: It the button's Y position. You think that's bottom?

**Pull request merged**